core/src/main/java/org/mobilitydata/gtfsvalidator/parsing/RowParser.java

```diff
@@ -137,6 +121,15 @@ public String asString(int columnIndex, GtfsColumnDescriptor columnDescriptor) {
               fileName, getRowNumber(), columnDescriptor.columnName()));
     }
     if (s != null) {
+      // Validate if the string contains invalid characters
+
+      if (containsInvalidCharacters(s)) {
+        noticeContainer.addValidationNotice(
+            new InvalidCharactersNotice(
+                fileName, getRowNumber(), columnDescriptor.columnName(), s));
+        return null;
```

Comment:
Let's remove the return statement and continue the validation below. This will allow offending feeds to pop up subsequence notices if they exist.

Reply:
Yes please! Ideally there are no dropped notices because of this new rule.

Reply:
it's removed.